spec/board_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'board'
+
+RSpec.describe Board do
+
+  it 'can take an array as an argument when initialised' do
```

Comment:
Ideally we should be testing behaviour, so I'm not sure this test adds much.

Reply:
Thanks, Matt